go.mod

```diff
@@ -35,16 +34,16 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/golang-lru v0.5.3
 	github.com/ishidawataru/sctp v0.0.0-20210226210310-f2269e66cdee // indirect
-	github.com/klauspost/compress v1.13.6
+	github.com/klauspost/compress v1.14.2
 	github.com/mitchellh/hashstructure/v2 v2.0.2
 	github.com/moby/locker v1.0.1
 	github.com/moby/sys/mount v0.3.0 // indirect
 	github.com/moby/sys/signal v0.6.0
 	github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6 // indirect
 	github.com/morikuni/aec v1.0.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.0.2-0.20210819154149-5ad6f50d6283
-	github.com/opencontainers/runc v1.0.3
+	github.com/opencontainers/image-spec v1.0.2-0.20211117181255-693428a734f5
+	github.com/opencontainers/runc v1.1.0
```

Comment:
`ARG RUNC_VERSION=v1.0.2` in Dockerfile should be updated too

Reply:
Argh looks like we need to build libseccomp 2.5.3 from source (latest on alpine is 2.5.2): https://github.com/moby/buildkit/runs/4977558239?check_suite_focus=true#step:6:271

```
#23 27.38 vendor/github.com/seccomp/libseccomp-golang/seccomp_internal.go:619:2: duplicate case _Ciconst_C_ACT_KILL_THREAD (value 0) in switch
#23 27.38 	previous case at vendor/github.com/seccomp/libseccomp-golang/seccomp_internal.go:615:2
#23 ERROR: process "/bin/sh -c CGO_ENABLED=1 xx-go build -mod=vendor -ldflags '-extldflags -static' -tags 'apparmor seccomp netgo cgo static_build osusergo' -o /usr/bin/runc ./ &&   xx-verify --static /usr/bin/runc" did not complete successfully: exit code: 2
```

Reply:
Actually that's an issue with the build tags for runc.

Reply:
Cannot a find a way a proper to build with gcc using xx if you have an idea @tonistiigi. Tried various combinations but xx always pick clang. I think for gcc we would need to define a pkgconfig with the proper flags in xx.